ci.sh

```diff
@@ -2,28 +2,17 @@
 
 echo "Installation of dependencies"
 
-# Install bleeding-edge dcm2niix
-curl -JLO https://github.com/rordenlab/dcm2niix/releases/latest/download/dcm2niix_lnx.zip
-unzip -o dcm2niix_lnx.zip
-sudo install dcm2nii* /usr/bin/
-
-echo "shimming-toolbox installation"
-pip install .
-pip install coveralls
-
-# Tests
-echo "Download prelude"
-st_download_data prelude
+# Dependencies
 echo "Set up prelude"
 prelude_path="$(pwd)"/prelude
-chmod 500 "${prelude_path}"/prelude
+chmod +x "${prelude_path}"/prelude
 PATH=${prelude_path}:${PATH}
```

Comment:
Ideally, prelude would exist as a standalone package we could use as a dependency, and installing it would look like installing dcm2niix. Why do you need to `+x` prelude but not dcm2niix?

Reply:
Can you use `install` on prelude instead?

Reply:
For sure, resolved in bd78ac6